packages/core/src/types/config/index.ts

```diff
@@ -180,7 +180,11 @@ export type ServerConfig<TypeInfo extends BaseKeystoneTypeInfo> = {
     app: express.Express,
     createContext: CreateRequestContext<TypeInfo>
   ) => void | Promise<void>;
-  extendHttpServer?: (server: Server, createContext: CreateRequestContext<TypeInfo>) => void;
+  extendHttpServer?: (
+    server: Server,
+    createContext: CreateRequestContext<TypeInfo>,
+    graphqlSchema: GraphQLSchema
```

Comment:
Could we not add this argument? you can already do `createContext().graphql.schema`